content/notebooks/data_discovery_and_access/data_discovery_and_access.ipynb

```diff
@@ -160,15 +164,15 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "uris = Observations.get_cloud_uris(single)\n",
+    "uris = Observations.get_cloud_uris(filtered)\n",
```

Comment:
I'm trying to run this command and I'm getting the following error message:
```
RemoteServiceError: Please enable anonymous cloud access by calling `enable_cloud_dataset` method. Refer to `~astroquery.mast.ObservationsClass.enable_cloud_dataset` documentation for more info.
```

Has this worked for you?

Reply:
This seems to be working for me when I run all the cells in the file both locally and on the science platform. Maybe there is something unique about your environment?